backend/de.metas.business/src/main/java/de/metas/order/impl/OrderLineBL.java

```diff
@@ -328,6 +328,12 @@ public void updateLineNetAmtFromQtyEntered(@NonNull final org.compiere.model.I_C
 		updateLineNetAmtFromQtyInPriceUOM(orderLine, qtyInPriceUOM);
 	}
 
+	@Override
+	public void updateMHuPiItemProduct(final I_C_OrderLine orderLine, final int huPiItemProductId)
```

Comment:
not really sure what the point of this method is..

Reply:
Me neither, removed.